### ISSUE TITLE
replacement

### ISSUE THREAD

**Issue by keradus** [Ocramius/FunctionFQNReplacer#6]
Hi !
Idea behind this package is really great!

This package is not actively maintained, and
some time ago, one of open-source contributor bring this feature into https://github.com/FriendsOfPHP/PHP-CS-Fixer/ as `native_function_invocation` rule.

Maybe you want to deprecate this package and redirect users to PHP CS Fixer to not need to maintain this package ?

**Comment by Ocramius** [comment]
Sure, just send a patch and we'll let it die 👍

**Comment by holtkamp** [comment]
👍 

**Off-topic / FYI**
Was curious about the benchmark, but (quick) installation seems to fail, the suggested:

`composer create-project roave/function-fqn-replacer`
results in

```
[InvalidArgumentException]                                                 
  Could not find package roave/function-fqn-replacer with stability stable.
```
and

```
git clone https://github.com/Roave/FunctionFQNReplacer
cd FunctionFQNReplacer
composer install
```

results in:

```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for humbug/humbug dev-master -> satisfiable by humbug/humbug[dev-master].
    - humbug/humbug dev-master requires phpunit/phpunit ^6.0 
```

**Comment by Ocramius** [comment]
yeah, humbug is currently in a "release limbo", where no existing version really works.

**Comment by keradus** [comment]
try https://github.com/infection/infection perhaps if you search for humbug replacement 
try PHP CS Fixer if you look for replacement of this package

**Comment by Ocramius** [comment]
Already fixed similarly via https://github.com/Ocramius/ProxyManager/pull/381 - just waiting for new PHPStan to finalize that and then we can apply it to various repos.
